lib/govuk_publishing_components/presenters/checkboxes_helper.rb

```diff
@@ -20,10 +20,13 @@ def initialize(options)
         @id = options[:id] || "checkboxes-#{SecureRandom.hex(4)}"
         @heading = options[:heading]
         @is_page_heading = options[:is_page_heading]
-        @hint_text = options[:hint_text] || "Select all that apply."
+        @no_hint_text = options[:no_hint_text]
+        @hint_text = options[:hint_text] || "Select all that apply." unless @no_hint_text
       end
 
       def fieldset_describedby
+        return if @no_hint_text
```

Comment:
I think this could be rewritten as:

```
unless @no_hint_text
   text = %w()
   text << "#{id}-hint" if @hint_text
   text << "#{id}-error" if @error
end
```

Reply:
@andysellick Sorry, my mistake for forgetting this when I copy/paste-d: you still need to return `text` before the `end`.